File: VB/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System.Web.Mvc

Namespace CS.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function All() As ActionResult
			Return View()
		End Function
		Public Function DX() As ActionResult
			Return View()
		End Function
		Public Function Plain() As ActionResult
			Return View()
		End Function
		Public Function SeparateAction() As ActionResult
			Return Content("Content From Separate <b>Action</b> Here...")
		End Function
	End Class
End Namespace

File: CS/Controllers/HomeController.cs
using System.Web.Mvc;

namespace CS.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }
        public ActionResult All() {
            return View();
        }
        public ActionResult DX() {
            return View();
        }
        public ActionResult Plain() {
            return View();
        }
        public ActionResult SeparateAction() {
            return Content("Content From Separate <b>Action</b> Here...");
        }
    }
}